apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -59,6 +62,7 @@ function WpcomNux() {
 		if ( isWpcomNuxEnabled && ! isSPTOpen ) {
 			recordTracksEvent( 'calypso_editor_wpcom_nux_open', {
 				is_gutenboarding: window.calypsoifyGutenberg?.isGutenboarding,
+				is_manually_opened: isGuideManuallyOpened,
```

Comment:
```suggestion
				is_manually_opened: !! isGuideManuallyOpened,
```

I think I've noticed the `is_manually_opened` prop can be missing from the event, and I think it's because the value defaults to `undefined`. Forcing the value to a boolean means it'll always send a `false` instead of just a falsey value.

Reply:
Ah, yes.  For the[ Tour, I set the default value in the store to false](https://github.com/Automattic/wp-calypso/blob/10cb217a9e8dc43f72cd70629cd195db062e60b5/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/store.js#L15), but I didn't do that for the NUX.    I had been debating refactoring the NUX store to be structured like the store in the Tour component, so I just went ahead and did that for clarity & consistency and[ set the default value for `isGuideManuallyOpened`](https://github.com/Automattic/wp-calypso/blob/4bcb955a8ab4329cba1b555319ac27c2ea003b6c/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/store.js#L15), which solves the issue you have pointed out. #:+1:

Reply:
Nice, what you've done is probably nicer tbh